### ISSUE TITLE
Telegram Bot

### ISSUE THREAD

**Issue by Sukriti-m** [Git21221/JS-beginner-projects#186]
I want to contribute a project which will be a telegram bot.

Tech Stack- JS

Please assign under hacktober 2023

**Comment by Git21221** [comment]
@Sukriti-m How it can be benificial for someone? Can they directly install the bot in their telegram?

**Comment by Sukriti-m** [comment]
Yes, I will provide a README.md file that will include the telegram bot name and then the user can search it's name in telegram and use it through various commands.

**Comment by Git21221** [comment]
@Sukriti-m I don't think I should include it in js project?

**Comment by Sukriti-m** [comment]
Well, it is mainly a NodeJS project. If you think it is not suitable for the repo then feel free to close the issue. I will raise issues only on Vanilla JS from now onwards.

**Comment by Git21221** [comment]
@Sukriti-m for now I am going ahead with your issue, but you can always create issue for another project on JS that should be suitable for this repo.

**Comment by Priya16042003** [comment]
please assign me the issue

**Comment by Priya16042003** [comment]
![image](https://github.com/user-attachments/assets/65f9a7f0-f600-410c-a4a8-f2e19d2374bd)
Pls checkout the telegram bot i have made it using python pls merge it
